rpc/src/main/scala/org/bitcoins/rpc/jsonmodels/WalletResult.scala

```diff
@@ -108,6 +110,18 @@ case class ReceivedAccount(
   lable: Option[String])
   extends WalletResult
 
+case class ReceivedLabel(
+  involvesWatchonly: Option[Boolean],
+  /**
```

Comment:
This was deprecated in `0.17` right? Technically I think we should use the `@deprecated` annotation on the method for now

Reply:
It's actually accounts that's deprecated, not labels. I was confused previously